### DISCUSSION TITLE
How releases and changelogs are generated for this project?

### DISCUSSION THREAD

**Discussion by dvakatsiienko** [react-hook-form/react-hook-form#6696] [Q&A] [answered]
**React hook form** releases and changelogs looks amazing.

What tools are used to setup those?
Probably there is a guide for that?
The description including links and emojies for each release generated automatically with some tool, or needs to be created manually using Github UI?

Thanks!

**Comment by bluebill1049** [comment, marked as answer]
hahaha thanks, I am manually taking care of those. 😆
